styles/Login.module.css

```diff
@@ -0,0 +1,61 @@
+.main {
+  min-height: 80vh;

```

Comment:
Just double check the responsiveness of this as vh depends on your viewport.

Reply:
Got it sir. I'll be converting to styled components so these styles are only temporary.